lua/lvim/core/cmp.lua

```diff
@@ -23,7 +23,7 @@ end
 M.methods.feedkeys = feedkeys
 
 ---when inside a snippet, seeks to the nearest luasnip field if possible, and checks if it is jumpable
----@param dir number 1 for forward, -1 for backward; defaults to 1
+---@param dir? number 1 for forward, -1 for backward; defaults to 1
```

Comment:
```suggestion
---@param dir number|nil 1 for forward, -1 for backward [default: 1]
```

Reply:
Is this a style preference to not use `?` but `number|nil`?

Reply:
> Is this a style preference to not use `?` but `number|nil`?

basically yes, at least that's the pattern I've seen preferred in neovim core.